<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrinterSettingsWidget</class>
 <widget class="QWidget" name="PrinterSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Menu_Control_LeftBlock">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>380</width>
     <height>540</height>
    </rect>
   </property>
   <property name="title">
    <string>Printer  Configs</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>374</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lb_Printer">
           <property name="text">
            <string>Printer:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_Printer"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="bt_SaveConfig">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bt_Default">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="Config">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="Connection_Config">
        <attribute name="title">
         <string>Connection</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="6" column="2">
          <widget class="QCheckBox" name="ck_ResetonEmergency">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lb_Protocol">
           <property name="text">
            <string>Firmware:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="4">
          <widget class="QComboBox" name="cb_Transmition_Rate">
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>57600</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1" colspan="4">
          <widget class="QComboBox" name="cb_Connection_Type">
           <property name="locale">
            <locale language="Portuguese" country="Brazil"/>
           </property>
           <item>
            <property name="text">
             <string>USB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Serial</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QCheckBox" name="ck_PrintLog">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QLabel" name="lb_Log">
           <property name="text">
            <string>Printer Log:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QCheckBox" name="ck_ResetonConnect">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lb_ResetC">
           <property name="text">
            <string>Reset on Connect:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lb_Cache">
           <property name="text">
            <string>Cache size:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lb_Port">
           <property name="text">
            <string>Port of Connection:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <widget class="QComboBox" name="cb_Connection_Port">
           <item>
            <property name="text">
             <string>/dev/ttyACM0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>/dev/ttyUSB0</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="1" colspan="4">
          <widget class="QComboBox" name="cb_Cache_Size">
           <item>
            <property name="text">
             <string>127</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lb_Transmission">
           <property name="text">
            <string>Transmission Rate:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="4">
          <widget class="QComboBox" name="cb_Firmware">
           <item>
            <property name="text">
             <string>Repetier</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Marlin</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lb_Connection">
           <property name="text">
            <string>Connection:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QLabel" name="lb_ResetE">
           <property name="text">
            <string>Reset on Emergency:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Print_Config">
        <attribute name="title">
         <string>Printer Specs</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="7" column="1" colspan="2">
          <widget class="QLineEdit" name="tb_AreaPrintY"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="lb_areaz">
           <property name="text">
            <string>Print Area - Vertical(mm)(Z):</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lg_Velocity_R">
           <property name="text">
            <string>Retraction Speed(mm/s):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lb_ExtruderV">
           <property name="text">
            <string>Extruder Speed(mm/s):</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lb_areax">
           <property name="text">
            <string>Print Area - Width(mm)(X):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="tb_RateMoviment"/>
         </item>
         <item row="8" column="1" colspan="2">
          <widget class="QLineEdit" name="tb_AreaPrintZ"/>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QLineEdit" name="tb_AreaPrintX"/>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="tb_FeedZ"/>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QLineEdit" name="tb_ExtTemperature"/>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="tb_RetractionSpeed"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lb_RateMoviment">
           <property name="text">
            <string>Moviment Rate(mm/min):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="tb_ExtSpeedmm"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lb_Feed_Z">
           <property name="text">
            <string>Feed Z Axis(mm/min):</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lb_TableTemperature">
           <property name="text">
            <string>Bed Temperature(ºC):</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="lb_areay">
           <property name="text">
            <string>Print Area - Height(mm)(Y):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="tb_ExtSpeedSec"/>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="QLineEdit" name="tb_BedTemp"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lb_Extruder_Temper">
           <property name="text">
            <string>Extruder Temperature(ºC):</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Extrusor_Config">
        <attribute name="title">
         <string>Extruder</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="6" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="images.qrc">:/Icons/Icons/bt-warningmax</pixmap>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QLabel" name="lb_obs">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">text-align: center;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Check with the manufacturer or the specifications of your&lt;br/&gt;printer to set Max temperatures on bed and (s) extruder(s). &lt;br/&gt;As for security, if any error occurs systemic or printer,&lt;br/&gt;these variables serve for emergency shutdown.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lb_extrudertmp">
           <property name="text">
            <string>Extruder Max Temperature(ºC):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="tb_ExtMaxTemp"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="ln_extruder_number">
           <property name="text">
            <string>Number of Extruders:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cb_ExtruderQnt"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lb_vol_sec">
           <property name="text">
            <string>Volume Max per Second(mm³/s):</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="tb_ExtMaxVol"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lb_tabletmp">
           <property name="text">
            <string>Bed Max Temperature(ºC):</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="tb_BedMaxTemp"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
